phpstan.neon

```diff
@@ -3,6 +3,10 @@ includes:
     - vendor/phpstan/phpstan/conf/bleedingEdge.neon
     - vendor/symplify/phpstan-extensions/config/config.neon
 
+    # see https://github.com/symplify/coding-standard
+    - vendor/symplify/coding-standard/config/symplify-rules.neon
+    - vendor/symplify/coding-standard/packages/cognitive-complexity/config/cognitive-complexity-services.neon
```

Comment:
temporary rollback register above rules as it seems the removal of rules rely to latest `dev-master` of symplify's coding-standard which rector uses released sympilfy's coding standard version.